src/main/java/com/thealgorithms/maths/DudeneyNumber.java

```diff
@@ -46,14 +48,24 @@ public static boolean isDudeney(int n) {
      * Number: 512 Output - It is a Dudeney Number. 2) Input - Enter a Number:
      * 125 Output - It is not a Dudeney Number.
      */
-    public static void main(String args[]) throws IOException {
+    /**
+     * Method to check if number is Dudeney Number or Not 1) Input - Enter a
+     * Number: 512 Output - It is a Dudeney Number. 2) Input - Enter a Number:
+     * 125 Output - It is not a Dudeney Number.
+     */
+    public static void main(String args[]) {
```

Comment:
Remove the main method as the tests already serve to demonstrate the algorithm works. By removing the main method the PR #3327 can be closed.

Reply:
Removed main method.